src/core/renderers/webgl/filters/SpriteMaskFilter.js

```diff
@@ -13,6 +12,7 @@ var AbstractFilter = require('./AbstractFilter'),
 function SpriteMaskFilter(sprite)
 {
     var maskMatrix = new math.Matrix();
+    var fs = require('fs');
```

Comment:
Not sure why browserify would fail, and others was me writing an example of how to do it and it got copy-pasted everywhere. fs should be required at the top of the file then used later on, not run in each constructor.

If there really is a browserify error, I'll fix it.